about.html

```diff
@@ -7,6 +7,7 @@
     <link rel="shortcut icon" href="assets/images/timer.png" type="image/x-icon">
     <link rel="stylesheet" href="assets/css/index.css">
     <link href='https://unpkg.com/boxicons@2.1.4/css/boxicons.min.css' rel='stylesheet'>
+    <script src="https://kit.fontawesome.com/d9b8043dd4.js" crossorigin="anonymous"></script>
```

Comment:
make use of boxicons and use icons from boxicons.com so that multiple CSS and scripts are not needed to use icons.

Reply:
Updated